src/util/invariant.cpp

```diff
@@ -116,20 +116,14 @@ void report_exception_to_stderr(const invariant_failedt &reason)
   std::cerr << "--- end invariant violation report ---\n";
 }
 
-std::string invariant_failedt::get_invariant_failed_message(
-  const std::string &file,
-  const std::string &function,
-  int line,
-  const std::string &backtrace,
-  const std::string &reason)
+std::string invariant_failedt::what() const noexcept
 {
   std::ostringstream out;
   out << "Invariant check failed\n"
-      << "File " << file
-      << " function " << function
-      << " line " << line << '\n'
-      << "Reason: " << reason
-      << "\nBacktrace:\n"
+      << "File " << file << " function " << function << " line " << line << '\n'
+      << "Condition: " << condition << '\n'
+      << "Reason: " << reason << '\n'
+      << "Backtrace:" << '\n'
       << backtrace << '\n';
```

Comment:
It seems that the word "Backtrace" occurs twice in the example, so I think we should remove
https://github.com/NlightNFotis/cbmc/blob/invariant_changes/src/util/invariant.cpp#L82

Reply:
Nice, I hadn't seen that backtrace was there. Thanks for it.